docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+**The number of available alternatives**
+
+Lately, with every new sunrise, there's a new UI component library. Naturally, considering the amount of iteration each one does on the same problems, patterns start to become more and more established. It's common to see libraries with very similar implementations of components, styling solutions, theming, etc. It gets harder to find ways to differentiate, to find something that would make one's iteration more compelling. That's a main reason we do this survey yearly, to shed light into improvements opportunities. Now that we have uncovered a bunch of spaces to explore, and together with our steadily growing team, we hope to focus on more than one scope at once with the same level of attention to detail and quality we've strived to have.
+
+**The introduction of a paid product**
+
+A big challenge with open-source is that the proportion of value captured by maintainers is close to zero. The MIT licensed model enabled a lot of growth but it naturally has reached a limit. In order to push our mission as much as we believe it's possible, we needed to introduce a way to capture more value. This was made by making MUI X open-core, instead of open-source, meaning having a full-time team of maintainers for the open-source and providing you with the most outstanding set of advanced components.
+
+It’s still the early days of MUI X, though. We introduced it at the end of 2020 and have been iterating on not only the components but on the model itself. We still have a lot more ground to cover, only ~0,1% of our developer community has been convinced to upgrade to the Pro plan so far. So, if you have more feedback about it, we'd be happy to hear. And if you want to understand more about our view of the MIT/open-source landscape, check our [Stewardship page](https://www.notion.so/Stewardship-542a2226043d4f4a96dfb429d16cf5bd).
+
+**The breaking changes made in v5**
+
+Our most recent release, MUI Core v5, introduced some important breaking changes. The main problem we wanted to tackle with it was customizability, something we have been hearing as an improvement opportunity for a while. The changes done are meant to unlock more styling capacity while not deprioritizing performance. We understand that it introduced a considerable amount of work to migrate but, we’re still in the early days of maturing this new styling solution. There are a lot of opportunities to expand on the reason why the change was made, how to get the most out of it, best practices, tutorials, automation tools, and, generally, how-to guide those concerned about migrating to v5 because of it. If you have any ideas or suggestions, you’re always welcome to share your thoughts by opening an issue on our repository.
```

Comment:
This is a very big block of text that could be split into several paragraphs for easier readability. When writing for online publications, I try to keep "paragraphs" to just 1, 2 or max 3 sentences. The goal is to make it as easy as possible to quickly skim.

```suggestion
Our most recent release, MUI Core v5, introduced some important breaking changes. The main problem we wanted to tackle was customizability: unlocking more styling options without compromising on performance. 

This new styling solution still has a ways to go in terms of maturity, and there's no denying that it requires a considerable amount of work to migrate. For our part, we still have a lot of work to do to help you make the most of these new features in v5, and we welcome your feedback. 

What can we do to help you succeed? If you have any ideas or suggestions, you’re always welcome to share your thoughts by opening an issue on our repository.
```